ui/lib/tokens/hooks/useValidVP.tsx

```diff
@@ -10,36 +10,36 @@ export function useValidVP(address: string | undefined) {
   const [L1Lock, setL1Lock] = useState()
   const [L2Lock, setL2Lock] = useState()
 
-  async function checkForLock() {
-    const L1sdk = initSDK(Ethereum)
-    const L2sdk = initSDK(Polygon)
-
-    const L1VMooneyContract = await L1sdk.getContract(
-      VMOONEY_ADDRESSES['ethereum']
-    )
-    const L2VMooneyContract = await L2sdk.getContract(
-      VMOONEY_ADDRESSES['polygon']
-    )
-
-    const L1Lock = await L1VMooneyContract.call('locked', [address])
-    const L2Lock = await L2VMooneyContract.call('locked', [address])
-
-    if (L1Lock[0] > 0 || L2Lock[0] > 0) {
-      setValidLock(true)
-    } else {
-      setValidLock(false)
+  useEffect(() => {
+    async function checkForLock() {
+      const L1sdk = initSDK(Ethereum)
+      const L2sdk = initSDK(Polygon)
+
+      const L1VMooneyContract = await L1sdk.getContract(
+        VMOONEY_ADDRESSES['ethereum']
+      )
+      const L2VMooneyContract = await L2sdk.getContract(
+        VMOONEY_ADDRESSES['polygon']
+      )
+
+      const L1Lock = await L1VMooneyContract.call('locked', [address])
+      const L2Lock = await L2VMooneyContract.call('locked', [address])
+
+      if (L1Lock[0] > 0 || L2Lock[0] > 0) {
+        setValidLock(true)
+      } else {
+        setValidLock(false)
+      }
+
+      const totalMooneyLocked = L1Lock[0]
```

Comment:
Ternary operators are really hard to parse and often prone to logical errors. Can you break this out into an if / else?

Reply:
Also, you're checking for L1Lock[0] and L2Lock[0] repeatedly. You could just store those as variables so you don't have to index into it repeatedly.

Reply:
Can do. Once we merge entity-registration into main I will deprecate this hook as we're now fetching voting power from snapshot.